pkgs/os-specific/linux/open-iscsi/default.nix

```diff
@@ -1,22 +1,23 @@
-{ stdenv, fetchurl, nukeReferences }:
+{ stdenv, fetchgit, nukeReferences, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod}:
 let
   pname = "open-iscsi-2.0-873";
```

Comment:
Done.

Any suggestion for an appropriate version string? 2.0-873 seems in appropriate. What about 2.0-873-4c1f2d?